src/classes/Cliente.java

```diff
@@ -0,0 +1,215 @@
+package classes;
+import excessoes.ExcessaoClienteMuitoAntigo;
+import excessoes.ExcessaoClienteMuitoNovo;
+import excessoes.ExcessaoDiaInvalido;
+import excessoes.ExcessaoEmailInvalido;
+import excessoes.ExcessaoMesInvalido;
+import excessoes.ExcessaoCPFInvalido;
+import excessoes.ExcessaoSomenteLetrasPermitidas;
+import excessoes.ExcessaoSomenteNumerosPositivosPermitidos;
+import excessoes.ExcessaoTelefoneInvalido;
+import excessoes.ExcessaoParametroVazio;
+
+import java.util.regex.Pattern;
+
+public class Cliente {
+	
+	private String CPF;
+	private String nome;
+	private String rua;
+	private String bairro;
+	private String cidade;
+	private String email;
+	private String telefone;
+	
+	private Data dataDeNascimento;
+	
+	private int numeroResidencia;
+	
+	private UnidadeFederativa UF;
+	
```

Comment:
Sobre a Classe Cliente:
Linha 19: Aqui poderia ter um atributo separado para número da rua, pra ter uma diferenciação do endereço.
Linha 21: Não sei se o campo cidade seria importante já que os clientes em sua maioria residem em campo grande.
Faltam os campos (baseado no mockup):
- RG

Reply:
Sobre a rua, não é o que está no slide, to seguindo o slide a risca, justamente pra não dar problema
Sobre o RG, não se aplica também pelo mesmo motivo, o que tem RG é o projeto principal esse é só o caso de uso CadastrarCliente dos slides